include/program_options_utils.hpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#ifndef DISKANN_PROGRAM_OPTIONS_UTILS_CPP
+#define DISKANN_PROGRAM_OPTIONS_UTILS_CPP
+
+#include <string.h>
+
+namespace program_options_utils
+{
+const std::string make_program_description(const char *executable_name, const char *description)
+{
+    return std::string("\n")
+        .append(description)
+        .append("\n\n")
+        .append("Usage: ")
+        .append(executable_name)
+        .append(" [OPTIONS]");
+}
+
+// Required parameters
+const char *DATA_TYPE_DESCRIPTION = "data type, one of {int8, uint8, float} - float is single precision (32 bit)";
+const char *DISTANCE_FUNCTION_DESCRIPTION =
```

Comment:
let's change it to distance function one of `{l2, mips, fast_l2, cosine}` ; we should just use `{item1, item2}` to indicate it's a set of values, and keep it consistent